drivers/common/mlx5/mlx5_prm.h

```diff
@@ -2081,6 +2081,106 @@ struct mlx5_ifc_query_virtq_out_bits {
 	struct mlx5_ifc_virtio_net_q_bits virtq;
 };
 
+
+struct regexp_params_field_select_bits {
+	u8         db_umem_id[0x1];
+	u8         stop_engine[0x1];
+	u8         reserved_at_2[0x1e];
+};
+
```

Comment:
small comment relevant for all of this patch. 
coding convention in DPDK there is no tab/space between the type and the field name.

Reply:
Do we have style checker script?